src/frontend/src/routes/(new-styling)/manage/(authenticated)/+layout.svelte

```diff
@@ -172,28 +170,15 @@
 
 <MainContent>
   {#snippet content()}
-    {#key page.url.pathname}
-      <div
-        bind:this={divRef}
-        in:fly={{
-          y: flyInY,
-          x: flyInX,
-          duration: 160,
-          delay: 160,
-          easing: expoOut,
-        }}
-        out:fly={{
-          y: flyOutY,
-          x: flyOutX,
-          duration: 160,
-          easing: expoIn,
-        }}
-        onoutrostart={() => divRef?.setAttribute("aria-hidden", "true")}
-        class="nth-2:hidden"
-      >
-        {@render children?.()}
-      </div>
-    {/key}
+    <div
+      bind:this={divRef}
+      in:fade={{ duration: 500 }}
```

Comment:
Since now we have no tabs or different ones, a fade in animation makes more sense.